sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java

```diff
@@ -12,7 +12,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-/** Aggregation that selects the specified default based on insturment. */
+/** Aggregation that selects the specified default based on instrument. */
```

Comment:
Actually it would be good to separate PRs for typo fixes and actual logic changes if that's OK

Reply:
Thanks @reugn - as this is small let me go ahead and merge, in the future would be good to split cosmetic and logic changes.